pandas/plotting/_core.py

```diff
@@ -308,10 +308,20 @@ def _setup_subplots(self):
 
         axes = _flatten(axes)
 
-        if self.logx or self.loglog:
+        valid_log = {False, True, 'sym', None}
+        input_log = {self.logx, self.logy, self.loglog}
+        if input_log - valid_log:
+            raise ValueError("Valid inputs are boolean, None and 'sym'.")
```

Comment:
Capture the bad parameter values and include them in the error message.

Reply:
thanks, changed! @TomAugspurger